scripts/healthomics_wf/create_healthomics_workflow.py

```diff
@@ -31,30 +31,39 @@ def localize_workflow(wf_root, aws_region, s3_bucket, aws_profile=None, input_te
     logging.info(f"workflow: {args.workflow} localization completed")
 
 
-def create_workflow(aws_region, omics_wf_name, wf_root, workflow_name):
-    create_omics_workflow(aws_region, omics_wf_name, wf_root, workflow_name)
+def create_workflow(aws_region, omics_wf_name, wf_root, workflow, aws_profile, use_dynamodb):
```

Comment:
Maybe I am missing something, but I don't see the value of this function, against directly calling `create_omics_workflow`